ports/raspberrypi/stage2.c.jinja

```diff
@@ -0,0 +1,197 @@
+#include "sdk/src/rp2040/hardware_structs/include/hardware/structs/ssi.h"
```

Comment:
I'm confused; I thought this file didn't fit in the 256 bytes, or did you remove some stuff or put it elsewhere?

Reply:
Removed the watchdog header.